selinux/ceph.te

```diff
@@ -88,6 +88,8 @@ logging_send_syslog_msg(ceph_t)
 
 sysnet_dns_name_resolve(ceph_t)
 
+apache_search_config(ceph_t)
```

Comment:
@alfredodeza sorry, i meant `/etc/httpd/`. i don't understand why ceph-mgr plugin reads this directory.

Reply:
@LenzGr @jcsp do you happen to know any of our plugins is reading from `/etc/httpd` for cert or some certain settings?

Reply:
Not that I know of (+ no hits for it grepping our source)

Reply:
@jcsp i also grep'ed the build directory and populated source tree. no luck.

Reply:
I'm also wondering why we'd need this.